File: LO52_A2019_BonsoirDAbord/TP1/HelloWorld/app/src/main/java/com/qcoudert/helloworld/MainActivity.java
package com.qcoudert.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    /**
     * Fonction onClick liée au bouton R.id.button
     * Lance l'activité {@link HelloWorldActivity} afin d'afficher "Hello World !"
     * @param v - View cliquée
     */
    public void onClickButton(View v) {
        Intent i = new Intent(getApplicationContext(), HelloWorldActivity.class);
        startActivity(i);
    }
}

File: tp1/src/main/java/fr/utbm/tp1/MainActivity.kt
package fr.utbm.tp1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.lo52.tp1.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onHelloWorldButtonClick(view: View){
        val intent = Intent(this, HelloWorldActivity::class.java)
        startActivity(intent)
    }
}
